### PR TITLE
Added BYN currency to MoneyData.csv

### PR DESCRIPTION
**Pull request by vytautasgimbutas**
Added BYN currency to MoneyData.csv. 
https://en.wikipedia.org/wiki/New_Belarusian_ruble

### CODE REVIEW

**Comment by vytautasgimbutas** [comment]
Could this be merged ASAP? In Android project I'm unable to add MoneyDataExtension.csv for some reason.

File: src/main/resources/org/joda/money/MoneyData.csv [hunk 1]
```diff
@@ -23,6 +23,7 @@ BRL,986,2,BR
 BSD,44,2,BS
 BTN,64,2,BT
 BWP,72,2,BW
+BYN,-1,2,BY
```
Comment (jodastephen) [inline src/main/resources/org/joda/money/MoneyData.csv]: should be `BYN,933,2,BY`
You also have to comment out `BYR` or the data won't load

Comment (vytautasgimbutas) [inline src/main/resources/org/joda/money/MoneyData.csv]: Why it won't load? Is it good idea to comment it out? I think there would be many cases where you might need to construct Money object with BYR currency.

I'll change -1 to 933.

Comment (vytautasgimbutas) [inline src/main/resources/org/joda/money/MoneyData.csv]: Now I see that there may not be multiple currency definitions for same country. Any ideas what to do if we still need to construct Money object with both currencies?

**Comment by jodastephen** [comment]
To have both BYR and BYN present, BYR must have no associated country code. However, this is not how the Joda-Money file is generally setup right now. Given this, the only PR I'd merge at Joda-Money would be one that replaces BYR with BYN.

Your best option to use both is to build this locally with your own data. Or to get `MoneyDataExtension.csv` working.

More generally, issue #64 needs to be resolved with a complete update of the data in this file.

**Comment by jodastephen** [comment]
Closing now that the data has been added (commenting out the old code)

**Pull request closed without merging**